src/frontend/context/DokumentutsendingContext.ts

```diff
@@ -52,8 +52,11 @@ export const [DokumentutsendingProvider, useDokumentutsending] = createUseContex
             verdi: Målform.NB,
         });
 
-        const årsak = useFelt<DokumentÅrsak>({
-            verdi: DokumentÅrsak.DELT_BOSTED,
+        const årsak = useFelt<DokumentÅrsak | undefined>({
```

Comment:
Dette er en ønsket endring fra Gunn: at vi ikke har en standardverdi for årsak men må gjøre et aktivt valg. Legger derfor til valideringen